src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+import core.basesyntax.controller.ConsoleHandler;
+import core.basesyntax.dao.BetDao;
+import core.basesyntax.exeptions.NonAnnotationException;
+import core.basesyntax.factory.Factory;
+import core.basesyntax.lib.Injector;
+import java.lang.reflect.InvocationTargetException;
+
+public class Main {
+    public static void main(String[] args) throws InvocationTargetException,
+            NoSuchMethodException, NonAnnotationException,
+            InstantiationException, IllegalAccessException {
+        ConsoleHandler handler = (ConsoleHandler) Injector
+                .getInstance(ConsoleHandler.class);
+        System.out.println("Введіть ваше ім'я, призвіще, "
+                + "сумму та ризик для вашої ставки через пробіл.\n"
+                + "Щоб завершити роботу програми натисніть 'q'.");
+        handler.handle();
+
+        BetDao betDao = Factory.getBetDao();
+        System.out.println(betDao.getAll());
```

Comment:
please remove this code from here
if you wish you can print out all the data in the end of your `handle()` method

Reply:
done